tox.ini

```diff
@@ -70,7 +70,7 @@ commands =
   nosetests -v --tests=test/visualiser
 
 [testenv:flake8]
-deps = flake8
+deps = flake8>=3.0.2
```

Comment:
I'm aware of it. While I never like to wake up to a red build status. It's not a bad thing to force ourselves to keep the code up to date to the latest regulations. :)